src/pages/User/components/PullRequests/UserInfo/MotivationalMessage/getMessage.js

```diff
@@ -14,21 +14,27 @@ const messages = [
   "Now you're just showing off!"
 ];
 
+const MONTH_OCTOBER = 9;
+
 /**
  * Get motivational message based on pull requests count.
  *
- * @param {*} pullRequestCount
- * @param {*} otherReposCount
+ * @param {integer} pullRequestCount
+ * @param {integer} otherReposCount
```

Comment:
JS doesn't have integer types. 
```suggestion
 * @param {number} pullRequestCount
 * @param {number} otherReposCount
```

Reply:
:ok: